script.js

```diff
@@ -0,0 +1,16 @@
+const hamburger = document.querySelector('#hamburger-icon');
+const cancelMenu = document.querySelectorAll('.cancel-menu');
+const nameTag = document.querySelector('#cancel-name-animation');
+const dropdown = document.querySelector('#dropdown-menu');
+
+hamburger.addEventListener('click', () => {
+  dropdown.classList.toggle('visibility');
+  nameTag.classList.toggle('pauseName');
+});
+
+for (let i = 0; i < cancelMenu.length; i += 1) {
+  cancelMenu[i].addEventListener('click', () => {
+    dropdown.classList.toggle('visibility');
+    nameTag.classList.toggle('pauseName');
+  });
+}
```

Comment:
- [ ] **In the README.md File:** Kindly make sure you add all the contributor's authors with their social links in your README to make it more professional.